### DISCUSSION TITLE
possibility to encrypt file

### DISCUSSION THREAD

**Discussion by Oops-Qiao** [idaholab/moose#23163] [Q&A General] [answered]
Hi,

I just have a quick question, is there a way in moose to hide some code or C file maybe?
:)

Thanks.
Qia

**Comment by Joseph-0123** [comment, marked as answer]
Hello @GiudGiud , could you please give us some tips on this? Thanks a lot.

**Reply by GiudGiud** [reply]
I have no idea how to do this. It’s not a common thing to do in our community, it’s pretty antithetical to open-source
